src/main/java/mate/academy/springboot/datajpa/repository/CategoryRepository.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.springboot.datajpa.repository;
+
+import mate.academy.springboot.datajpa.model.Category;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface CategoryRepository extends JpaRepository<Category, Long> {
+    @Modifying
+    @Query("update Category c set c.name = ?2 where c.id = ?1")
+    void update(Long id, String name);
```

Comment:
Do you really need this? Just put id after you map dto to model in controller or in service and use method from box